### PR TITLE
Add html styleguide

### PR DESCRIPTION
**Pull request by acacheung**

### CODE REVIEW

**Comment by acacheung** [comment]
@Ctannerweb :eyes: plz

**Comment by Ctannerweb** [comment]
@acacheung  :+1: Will do

File: uxd/html.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# HTML
+
+Read and follow the
+[HTML section]
+(http://codeguide.co/#html)
+of Mark Otto’s code guide.
+
+In addition to @mdo’s Code Guide:
```
Comment (Ctannerweb) [inline uxd/html.md]: Needs link to @mdo's Code Guide http://codeguide.co/

File: uxd/html.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+## Indentation and Line Breaks
+
+Break to a new line the tag contains another element.
```
Comment (Ctannerweb) [inline uxd/html.md]: Break to a new line if the tag contains another element.

File: uxd/html.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+Bad:
+
+    <p>This is a <a href="#">link</a>.</p>
+
+Good:
+
+    <p>
+      This is a
+      <a href="#">link</a>.
```
Comment (Ctannerweb) [inline uxd/html.md]: is the . a typo??

File: uxd/html.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+[extra space]
+(http://stackoverflow.com/questions/588356/why-does-the-browser-renders-a-newline-as-space)
+when the browser renders it.
+
+This also makes sense:
+
+    <h2>June 16<sup>th</sup></h2>
+
+because there shouldn't be a space or possibility of line break
+between the date and its ordinal indicator.
```
Comment (Ctannerweb) [inline uxd/html.md]: Link to what a ordinal indicator is, http://en.wikipedia.org/wiki/Ordinal_indicator

File: uxd/html.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+## Comments
```
Comment (Ctannerweb) [inline uxd/html.md]: Add code examples for each comment type

File: uxd/html.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+* [Use escapes for `&lt;`, `&gt;`, `&amp;`]
+(http://www.w3.org/International/questions/qa-escapes#use).
+
+* No need to escape for
+[smartquotes]
+(http://smartquotesforsmartpeople.com/).
+
+## QA
+
+1. Check html in an outliner. Does it makes sense?
```
Comment (Ctannerweb) [inline uxd/html.md]: Give link to outliner you want used. WAVE??
https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US

**Pull request merged**